.github/workflows/test-gradle.yml

```diff
@@ -16,6 +16,9 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    env:
+      GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
```

Comment:
this is not our main build - our main builds are on https://ci-builds.apache.org/job/POI/

this Github CI build only kicks off for commits by ASF committers 

we use svn - not github - this is a mirror over which we have little github control